modules/core/04-channel/types/msgs_test.go

```diff
@@ -235,7 +230,6 @@ func (suite *TypesTestSuite) TestMsgChannelOpenConfirmValidateBasic() {
 		{"too long channel id", types.NewMsgChannelOpenConfirm(portid, invalidLongChannel, suite.proof, height, addr), false},
 		{"channel id contains non-alpha", types.NewMsgChannelOpenConfirm(portid, invalidChannel, suite.proof, height, addr), false},
 		{"empty proof", types.NewMsgChannelOpenConfirm(portid, chanid, emptyProof, height, addr), false},
-		{"proof height is zero", types.NewMsgChannelOpenConfirm(portid, chanid, suite.proof, clienttypes.ZeroHeight(), addr), false},
```

Comment:
should we maybe modify one of these test cases to explicitly state that the test should pass with a zero height? Rather than just removing the expected failures?

Reply:
Using the tendermint clients on the test suite chains the code flow for connection/channel handshakes would still likely fail [here](https://github.com/cosmos/ibc-go/blob/main/modules/light-clients/07-tendermint/client_state.go#L237), as there would be no consensus state stored for the zero height afaik.

Reply:
I see thanks!